lib/command.js

```diff
@@ -265,6 +270,10 @@ class Command extends EventEmitter {
       throw new Error(`Command passed to .addCommand() must have a name
 - specify the name in Command constructor or using .name()`);
     }
+    if (cmd.parent) {
+      throw new Error(`'${cmd._name}' cannot be added using .addCommand()
```

Comment:
I quite like the format of the error just above. Maybe:
```
throw new Error(`Command '${cmd.name()}' passed to .addCommand() already has a parent
 - commands can not be reused or shared`);
```

Reply:
I like it, but I would leave out "reused" because "shared" means the same thing here but is more specific. I would maybe change "commands" to "subcommands" on the second line, too.

Reply:
Yup, I like both.

Reply:
Resolved in https://github.com/tj/commander.js/pull/1917/commits/55b9146cbf23fc20e9b32bedadb27d2169ee41a1.